pkg/systemd/quadlet/quadlet.go

```diff
@@ -672,6 +674,12 @@ func ConvertContainer(container *parser.UnitFile, names map[string]string, isUse
 		return nil, err
 	}
 
+	groupsAdd := container.LookupAll(ContainerGroup, KeyGroupAdd)
+	for _, groupAdd := range groupsAdd {
+		if len(groupAdd) > 0 {
+			podman.add("--group-add", groupAdd)
+		}
+	}
```

Comment:
Sorry for the nit. Can you please add an empty line here

Reply:
Done.